src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Tests/System.Text.Json.SourceGeneration.Tests.targets

```diff
@@ -22,6 +22,10 @@
     <DefineConstants>$(DefineConstants);BUILDING_SOURCE_GENERATOR_TESTS</DefineConstants>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Please remove this dependency, while FluentAssertions are a good extension it would preferable if you could follow the conventions used by the rest of the test suite.